<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setupAPI</class>
 <widget class="QDialog" name="setupAPI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>137</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setup api.ini</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="apiFileLocationDisplay">
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="selectAPIFileButton">
     <property name="text">
      <string>Select API File</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="addAPIEntryButton">
     <property name="text">
      <string>Add Listener to Hex's api.ini</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
